tests/rustdoc/doc-cfg-unstable.rs

```diff
@@ -0,0 +1,11 @@
+//@ should-fail
+
```

Comment:
Would be good to actually check that it fils because the missing feature.

Shouln't this test also be a rustdoc UI test?